### DISCUSSION TITLE
Firefly incorrect time zone

### DISCUSSION THREAD

**Discussion by 0styx0** [firefly-iii/firefly-iii#7825] [Firefly III] [answered]
Firefly displays an incorrect time zone.

Example steps to see the incorrect time zone:
- Create a new withdrawal
- The time in the transaction will display as if the time-zone is set to UTC

Extra information: I installed firefly through docker-compose

### Debugging steps I've taken
- Verified `timedatectl` returns the time-zone in EDT on my host machine
    - This step shouldn't matter, but listing it anyways
- Set `TZ` to `TZ=US/Eastern` in my `.env`
- Ran a `docker exec -it <docker_container_name> /bin/bash`
    - `echo $TZ` --> `US/Eastern`
    - Ensured `US/Eastern` exists: `ls /usr/share/zoneinfo/US/Eastern`
- Ran an `ls /etc/localtime -l`. Output: Symlinked to `/usr/share/zoneinfo/Etc/UTC` in the docker container
    - Ran a `ln -s /usr/share/zoneinfo/US/Eastern /etc/localtime` and verified `/etc/localtime` then symlinked to `US/Eastern`
- Ran a `docker-compose up -d`
- Rebooted my host machine
- Went back to fireflyiii between each bullet and refreshed by browser cache via `<C-S-R>` and also cleared firefly cache in `Options -> administration -> Maintenance`
- Browsed similar discussions and the internet. I've only seen answers involving setting the `TZ` environment variable

Nothing I've tried has changed the time-zone behavior. Help would be greatly appreciated :)

**Comment by JC5** [comment]
What happens when you run `docker exec -it [container] php artisan config:clear`?

**Reply by 0styx0** [reply]
No change.

Figured out the issue, though. I set `privacy.resistFingerprinting` in Firefox's `about:config`

**Reply by 0styx0** [reply, marked as answer]
Fix: Set `privacy.resistFingerprinting.exemptedDomains` to the domain of firefly

Thank you for your time
